projects/wave-core-new/src/lib/layers/legend/legend-vector/vector-legend-component.html

```diff
@@ -1,13 +1,87 @@
+            <span fxFlex
+                >Fill Colors derived by <b><i>{{ colorAttributeName }}</i></b></span
+            >
+        </div>
+        <div fxLayout="row" fxLayoutAlign="start center" class="segment" *ngFor="let breakpoint of colorBreakPoints">
+            <mat-icon>
+                <ng-container [ngSwitch]="layer.symbology.symbologyType">
+                    <wave-point-icon
+                        *ngSwitchCase="ST.POINT"
+                        [iconStyle]="getPointIconStyle({fillColor: breakpoint.color})"
+                    ></wave-point-icon>
+                    <wave-polygon-icon
+                        *ngSwitchCase="ST.POLYGON"
+                        [iconStyle]="getPolygonIconStyle({fillColor: breakpoint.color})"
+                    ></wave-polygon-icon>
+                </ng-container>
+            </mat-icon>
+            <span fxFlex>{{breakpoint.value}}</span>
+        </div>
+    </div>
+
+    <!-- Stroke Color Legend   -->
+    <div *ngIf="strokeColorBreakpoints">
+        <mat-divider></mat-divider>
+        <div fxLayout="row" fxLayoutAlign="start center" class="heading">
+            <span fxFlex
+                >Stroke Colors derived by <b><i>{{ strokeColorAttributeName }}</i></b></span
+            >
+        </div>
+        <div fxLayout="row" fxLayoutAlign="start center" class="segment" *ngFor="let breakpoint of strokeColorBreakpoints">
+            <mat-icon>
+                <ng-container [ngSwitch]="layer.symbology.symbologyType">
+                    <wave-point-icon
+                        *ngSwitchCase="ST.POINT"
+                        [iconStyle]="getPointIconStyle({strokeWidth: 1, strokeColor: breakpoint.color})"
+                    ></wave-point-icon>
+                    <wave-polygon-icon
+                        *ngSwitchCase="ST.POLYGON"
+                        [iconStyle]="getPolygonIconStyle({strokeWidth: 1, strokeColor: breakpoint.color})"
+                    ></wave-polygon-icon>
+                </ng-container>
+            </mat-icon>
+            <span fxFlex>{{breakpoint.value}}</span>
+        </div>
+    </div>
+
+    <!-- Stroke Width Legend   -->
+    <div *ngIf="strokeWidthAttributeName">
+        <mat-divider></mat-divider>
+        <div fxLayout="row" fxLayoutAlign="start center" class="heading">
+            <span fxFlex
+                >Stroke Width derived by <b><i>{{ strokeWidthAttributeName }}</i></b></span
+            >
+        </div>
+        <div fxLayout="row" fxLayoutAlign="start center" class="segment">
+            <span fxFlex> with factor: {{ strokeWidthFactor }}</span>
+        </div>
+    </div>
+
+    <!-- Radius Legend  ??? -->
+    <div *ngIf="radiusAttributeName">
+        <mat-divider></mat-divider>
+        <div fxLayout="row" fxLayoutAlign="start center" class="single-line">
+            <span fxFlex
+                >Radius derived by <b><i>{{ radiusAttributeName }}</i></b></span
+            >
```

Comment:
I guess you can also add the factor here like for stroke width.

Reply:
I added the factor of the radius as well: 3d5f966e761ff9e0888ee8ff23d48968641d47e2